gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -654,52 +707,22 @@ def _import_related_samples_to_drop(**kwargs):
     CURRENT_VERSION,
     {
         version: TableResource(
-            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.regressed_metrics.ht"
-        )
-        for version in VERSIONS
-    },
-)
-
-# Ranking of all samples based on quality metrics. Used to remove relateds for PCA.
-pca_samples_rankings = VersionedTableResource(
-    CURRENT_VERSION,
-    {
-        version: TableResource(
-            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.pca_samples_ranking.ht"
-        )
-        for version in VERSIONS
-    },
-)
-
-# Ranking of all release samples based on quality metrics. Used to remove relateds for
-# release.
-release_samples_rankings = VersionedTableResource(
-    CURRENT_VERSION,
-    {
-        version: TableResource(
-            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.release_samples_ranking.ht"
+            f"{get_sample_qc_root(version)}/outlier_detection/gnomad.exomes.v{version}.regressed_metrics.ht"
         )
         for version in VERSIONS
     },
 )
 
-# Duplicated (or twin) samples.
-duplicates = VersionedTableResource(
-    CURRENT_VERSION,
-    {
-        version: TableResource(
-            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.duplicates.ht"
-        )
-        for version in VERSIONS
-    },
-)
 
-# v4 samples that failed fingerprinting.
-fingerprinting_failed = VersionedTableResource(
+######################################################################
+# Other resources
+######################################################################
+# Number of clinvar variants per sample.
+sample_clinvar_count = VersionedTableResource(
     CURRENT_VERSION,
     {
         version: TableResource(
-            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.fingerprintcheck_failures.ht"
+            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.clinvar.ht"
```

Comment:
Everything else, with the exception of `pre_ld_pruning_qc_variants.ht` are in subdirectories, how do you feel about making another subdirectory under gnomad/v4.0/sample_qc of `additional_resources` or some better name?